client/src/components/SolutionsSectionHeader.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="mj-solution-section-header">
+    <mj-icon
+      :name="name"
+      class="mj-solution-section-header-icon" />
+    <h1>
+      <slot />
+    </h1>
+    <router-link
+      to="/journal/mistakes/"
+      tag="a"
+      role="button"
+      class="btn with-icon mj-solution-section-header-close">
+      <fa-layers>
+        <fa-icon :icon="['fas', 'times']" transform="grow-3"/>
+        <fa-icon :icon="['far', 'circle']" transform="grow-13"/>
+      </fa-layers>
+    </router-link>
+  </div>
+</template>
```

Comment:
I see that you added new "SectionHeader". TBH I would consider extending already existing SectionHeader (because this one is a copy, we try to reuse components instead of copying them). You can edit SectionHeader and add something called named slot: https://vuejs.org/v2/guide/components-slots.html#Named-Slots
I would do is like this in SectionHeader:
```
<template>
  <div class="mj-section-header">
    <mj-icon
      :name="name"
      class="mj-section-header-icon" />
    <h1>
      <slot />
    </h1>
    <div class="mj-section-header-actions">
        <slot name="actions" />
    </div>
  </div>
</template>
```

And with something like that, you can just call it in MistakesPage:
```
<solutions-section-header name="tips">
  {{ $t('Solutions.Title') }}
  <template v-slot:header>
    <router-link
      to="/journal/mistakes/"
      tag="a"
      role="button"
      class="btn with-icon mj-solution-close">
      <fa-layers>
        <fa-icon :icon="['fas', 'times']" transform="grow-3"/>
        <fa-icon :icon="['far', 'circle']" transform="grow-13"/>
      </fa-layers>
    </router-link>
  </template>
</section-header>
```
```

Reply:
done :)